ui/.neutrinorc.js

```diff
@@ -60,20 +60,20 @@ module.exports = {
         },
         proxy: {
           '/login': {
-            target: 'http://localhost:3050',
+            target: 'https://community-tc.services.mozilla.com/',
             changeOrigin: true,
           },
           '/graphql': {
-            target: 'http://localhost:3050',
+            target: 'https://community-tc.services.mozilla.com/',
             changeOrigin: true,
           },
           '/subscription': {
             ws: true,
             changeOrigin: true,
-            target: 'ws://localhost:3050',
+            target: 'https://community-tc.services.mozilla.com/',
           },
           '/api/web-server': {
-            target: 'http://localhost:3050',
+            target: 'https://community-tc.services.mozilla.com/',
```

Comment:
As @djmitche pointed out, it's better not to change those lines here. I only suggested adding those to skip running other services locally. 

We would definitely address it in a separate issue. I think we can support `TASKCLUSTER_ROOT_URL` variable here, but it will go in a new issue.

Reply:
Okay. I understannd @lotas 
Thanks

Reply:
Added new ticket, linking here for the records https://github.com/taskcluster/taskcluster/issues/5373